doc/manual/preprocessing/ssp.rst

```diff
@@ -126,3 +115,77 @@ decomposition, and employ the eigenvectors corresponding to the
 highest eigenvalues as basis for the noise subspace. It is also
 customary to use a separate set of vectors for magnetometers and
 gradiometers in the Vectorview system.
+
+Average reference
+-----------------
+
+The EEG average reference is the mean signal over all the sensors. It is typical in EEG analysis to subtract the average reference from all the sensor signals :math:`b^{1}(t), ..., b^{n}(t)`. That is:
+
+.. math::	{b}^{j}_{s}(t) = b^{j}(t) - \frac{1}{n}\sum_{k}{b^k(t)}
+   :label: eeg_proj
+
+where the noise term :math:`b_{n}^{j}(t)` is given by
+
+.. math:: 	b_{n}^{j}(t) = \frac{1}{n}\sum_{k}{b^k(t)}
```

Comment:
same with this. A second eye might be useful. The MNE manual says `\frac{1}{\sqrt{n}}` but I don't follow and the code also doesn't seem to say something different (https://github.com/mne-tools/mne-python/blob/master/mne/io/proj.py#L639)